.github/workflows/ci.yml

```diff
@@ -82,9 +82,7 @@ jobs:
         name: ${{ runner.os }}-native-library
```

Comment:
For some reason, GitHub does not allow to comment the code above your PR. Nonetheless, I think now that the `dotnet test` at line 78 is redundant (as well as the step name). We should probably replace it with `dotnet build`.

Reply:
Good point. Maybe we go one step further and only build the native library? I think there is no reason to build the .NET assembly anymore, and it would also respect the job name (`build-native`).

Reply:
Was wondering the same thing.

It has the advantages of checking whether the whole repository compiles out of the box. Because we do quite a few manipulations on `csharp.test` in the `test-nuget` job, it's quite possible for the whole repository to be broken for local builds and we would never notice (e.g. missing dependencies on `csharp`, build order, etc).

I.e. this tests the developer workflow. While the jobs below test the user (nuget) worflow.

Reply:
Agreed, let's keep it as-is then.